### DISCUSSION TITLE
Cinlerin evine ziyaret sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#99] [Zeka Sorusu]
3 tane cinin evini ziyarete gidiyorsunuz, bunların 3' ü de kırmızı gözlü. ama 3' ü de kendi göz rengini bilmiyor ve göz renkleri hakkında hiç konuşmuyorlar. evde ayna ve benzeri bir cisim de yok. evin bir kuralı var; bir cin kırmızı gözlü olduğunu bilirse ilk güneş doğuşunda ölüyor. bu konu hakkında hiç konuşmadıkları için kimse kendi göz renginden emin olamıyor ve mutlu mesut yaşıyorlar.
evden ayrılmadan cinlerin işine karışmaya karar veriyorsunuz ve "en az biriniz kırmızı gözlü" bilgisini veriyorsunuz. bu bilgi evin gidişatında bi değişiklik yapar mı? yaparsa cinler kaç güneş doğuşu daha görürler?

**Comment by Eceg25** [comment]
Zaten bunu söylemeden öncede birbirlerinin göz renklerini görüyorlar ve hepsi diğer iki cinin kırmızı gözlü olduğunu bildiği halde bundan bahsetmiyorlar. Bu konu hakkında konuşmadıkları için bizim bunu söylememiz bir şeyi değitirmez.

**Reply by Onuralp000** [reply]
cinler yine göz renklerini bilmemeye devam ederler. Dolayısıyla, cinler kaç güneş doğuşu daha gördüklerinde değişmez ve aynı kurallar geçerli olur.

**Comment by Efemert2010** [comment]
Eğer benim gözlerim kırmızı değilse, karşımdaki diğer iki cin birbirlerinin kırmızı gözlü olduğunu göreceklerdir ve bu durumda ilk güneş doğuşunda öleceklerdir. Çünkü eğer sadece onlar kırmızı gözlü olsalardı, bunu hemen fark eder ve ölürlerdi.
Ancak ilk güneş doğuşunda hiç kimse ölmeyince, her biri kendi gözlerinin de kırmızı olduğunu anlar. Çünkü eğer gözleri kırmızı olmasaydı, diğer iki cin birbirlerinin gözlerinin kırmızı olduğunu fark edecek ve bir önceki güneş doğuşunda öleceklerdi.
İlk gün, her cin diğer iki kırmızı gözlü cini görür ve kendi gözlerinin kırmızı olmadığını düşünürse, diğer iki cinin birbirlerini görerek ilk gün öleceklerini düşünür.
Ancak ilk gün kimse ölmez. İkinci güneş doğuşuna kadar hayatta kalırlar.
İkinci güneş doğuşunda ise her biri, kendi gözlerinin de kırmızı olduğunu fark eder. Çünkü eğer gözleri kırmızı olmasaydı, diğer iki cin ilk güneş doğuşunda ölecekti.
Sonuç olarak, cinler ikinci güneş doğuşunda ölür. Yani toplamda bir güneş doğuşu daha görürler.

**Comment by Productiveboy** [comment]
Eğer cinler biribirinin kırmızlı gözlü olduğunu söylerse ve sonuç birine varırsa bu cin bir diğer güneş doğuşunda ölür. Veya herkes kendinin kırmızı gözlü olmadığından emin olmak için ayna vs. bakarlarsa bu onların bir tanesinin ölümüyle sonuçlanır.Ama zaten bahsetmedikleri için pek bir sorun da olmayabilir.

**Comment by Sevvalgocgun1** [comment]
Cinler 2 güneş doğuşu daha görürler.
